gravitee-apim-console-webui/src/management/api/documentation-v4/api-documentation-v4.component.ts

```diff
@@ -79,6 +79,7 @@ export class ApiDocumentationV4Component implements OnInit, OnDestroy {
             parentId: this.parentId,
           }),
         ),
+        switchMap((createdFolder) => this.apiDocumentationV2Service.publishDocumentationPage(this.ajsStateParams.apiId, createdFolder.id)),
```

Comment:
Suggestion: what about keeping this logic on the back: when creating a folder, this one could be marked as published in the creation process ?

Reply:
I think I prefer to keep the create and publish separate, because if they're combined and a user wants to use the backend only, they would be obliged to publish a folder every time they create. The automatic publishing of folders in the console is just so that it appears in Portal right away.